docs/source/release/v6.3.0/muon.rst

```diff
@@ -9,4 +9,15 @@ MuSR Changes
     putting new features at the top of the section, followed by
     improvements, followed by bug fixes.
 
+Muon Analysis and Frequency Domain Analysis
+-------------------------------------------
+
+New Features
+############
+- Added a :ref:`Power Law <func-PowerLaw>` function to MuonModelling Fit Functions.
```

Comment:
also worth putting in the framework (see my note above about adding it to general)